src/client/components/robot_selector/robot_label/format-si.ts

```diff
@@ -0,0 +1,48 @@
+// Adapted from https://github.com/ThomWright/format-si-prefix
+
+const prefixes: { [key: string]: string } = {
+  '24': 'Y',
+  '21': 'Z',
+  '18': 'E',
+  '15': 'P',
+  '12': 'T',
+  '9': 'G',
+  '6': 'M',
+  '3': 'k',
+  '0': '',
+  '-3': 'm',
+  '-6': 'µ',
+  '-9': 'n',
+  '-12': 'p',
+  '-15': 'f',
+  '-18': 'a',
+  '-21': 'z',
+  '-24': 'y',
+}
+
+export function formatSI(value: number) {
```

Comment:
This looks like something easily testable 🙂

Reply:
:grinning: True. I'll add some tests...

Reply:
Added in https://github.com/NUbots/NUsight2/pull/269/commits/f68f92f91a4c839951594c6c47cfbfcb240f09a1